include/sas.h

```diff
@@ -104,6 +104,7 @@ enum struct UniquenessScopes
   UUID_RFC4122 = 2,
   ASYNC_CFG_SUB_DN = 3,
   DIGEST_OPAQUE = 4
+  STEERING_ID = 5
```

Comment:
Don't you need a comma on the previous line?

Reply:
Thanks, changed.